java/kotlin-extractor/BUILD.bazel

```diff
@@ -30,9 +30,9 @@ version. Possible workarounds for that:
 """
 
 # This file is used in the `@codeql_kotlin_embeddable` external repo, which means we need to
-# reference explicitly @codeql
+# reference explicitly @ql
 load(
-    "@codeql//java/kotlin-extractor:versions.bzl",
+    "@ql//java/kotlin-extractor:versions.bzl",
```

Comment:
There was some `@codeql`s in the comments at the start of this file; presumably they need to be changed too?

Reply:
I think I changed the relevant ones - when you build from the internal repo, we have an alias in place that you can still call the build targets @codeql//something, only from the external repo it's now (unfortunately) @ql//something, which is why I changed the wording of the comment there to remove the repo name alltogether when building from the external repo.
That said, there might still be an incorrect comment there, so if you find one, please shout!